app/models/programme_activity_grouping.rb

```diff
@@ -2,6 +2,8 @@ class ProgrammeActivityGrouping < ApplicationRecord
   has_many :programme_activities, -> { order(:order) }
   belongs_to :programme
 
+  scope :progress_bar_grouping, -> { where.not(progress_bar_title: nil) }
+  
```

Comment:
One recommendation for the future (but not to delay this PR) is to name scopes with plurals, as they return a collection, so  `progress_bar_groupings` for example. But you'll have to change everywhere you use it and ask me or Caleb to review again, so don't do it here.

Reply:
I only use it in 2 places so this is a quick change thank you